### DISCUSSION TITLE
百度网盘大文件无法上传

### DISCUSSION THREAD

**Discussion by xuange-dada** [AlistGo/alist#2787] [General]
![电脑录屏快捷键win10 - Google 搜索 - Google Chrome 2022_12_23 0_20_03](https://user-images.githubusercontent.com/70432769/209178093-3cf567de-4053-4af1-8983-2da418d21054.png)

**Comment by xuange-dada** [comment]
有大佬懂吗，小于20G（18G左右）的文件依旧无法上传

**Reply by anwen-anyi** [reply]
老老实实用百度云官方的上传吧QAQ 这么大

**Reply by xuange-dada** [reply]
> 老老实实用百度云官方的上传吧QAQ 这么大

主要是没会员想上传超过20G限制的文件，api接口貌似没有限制，但是对于小于20G的大文件能在客户端上传在alist网页却报错
